scripts/DBAG-JP/c100416033.lua

```diff
@@ -122,6 +122,8 @@ function c100416033.thcost(e,tp,eg,ep,ev,re,r,rp,chk)
 end
 function c100416033.thtg(e,tp,eg,ep,ev,re,r,rp,chk)
 	if chk==0 then return Duel.IsExistingMatchingCard(c100416033.tgfilter,tp,LOCATION_GRAVE,0,1,nil,e,tp,true) end
+	Duel.SetOperationInfo(0,CATEGORY_TOHAND,nil,1,tp,LOCATION_GRAVE)
+	Duel.SetOperationInfo(0,CATEGORY_SPECIAL_SUMMON,nil,1,tp,LOCATION_GRAVE)

```

Comment:
for not certain action, `SetOperationInfo` shouldn't be used

Reply:
but, it can be negated by NecroValley, is this like _Lazion, the Timelord_?

Reply:
use `aux.NecroValleyFilter`

Reply:
this effect only moves a card from the grave, i think it will be negated rather than use `aux.NecroValleyFilter`